Makefile

```
@@ -19,20 +19,27 @@ help:
 
 .PHONY: dev
 dev: python
-	@tox -qe dev
+	@tox -qe dev -- honcho start
 
 .PHONY: devdata
 devdata: python
 	@tox -qe dev -- python bin/devdata.py
 
+.PHONY: web
+web: python
+	@tox -qe dev
+
+.PHONY: nginx
+nginx: python
+	@tox -qe docker-compose up
```

It would be nice to support some of the same semantics as usual `make service` here. I think not detaching is actually fine, but it would be nice to default the args as we usually do, so they can be overridden for things like `make nginx args=down`

```suggestion
.PHONY: nginx
nginx: args?=up
nginx: python
	@tox -qe docker-compose -- $(args)
```

---

Done this